### ISSUE TITLE
configs missing paramter when it is in grid auto gen/split

### ISSUE THREAD

**Issue by madragonse** [llm-random/nano#35]
in configs:

 ^learning_rate: 
     - 3.0517578125e-05
    # - 6.103515625e-05
    # - 1.52587890625e-05

gives missing learning rate exception when parsing onfig, unexpected behaviour, prviousy this worked i think, maybe learning_rate: ??? breakes it in inhereted template configs?
